### PR TITLE
reorganized install/run/config in main vignette

### PR DESCRIPTION
**Pull request by JSchripsema-IDM**

### CODE REVIEW

File: pacehrh/vignettes/pacehrh.rmd [hunk 1]
```diff
@@ -15,15 +15,81 @@ knitr::opts_chunk$set(echo = TRUE)
...
+
+<!--- Can this only be run on Windows? Do we have alternate directions for Mac or Linux OS?--->
+
+### Install using the current released version Windows binaries 
+
+Follow the instructions below to install a tested, released version of the model.
+
+1.  Download the latest binary zip file from https://github.com/InstituteforDiseaseModeling/PACE-HRH/releases.
+    This will be the first file listed under **Assets**. 
+    ![Binary zip file under GitHub release assets.](images/pacehrh-release.png)
```
Comment (MeWu-IDM) [inline pacehrh/vignettes/pacehrh.rmd:42]: Was this image checked in? I did not see it

**Review by MeWu-IDM — CHANGES_REQUESTED** [review]
The description looks good to me, but I did not see the image mentioned attached, 
also as of our current release process, in order to publish this we need to bump the version 
in DESCRIPTION file

File: pacehrh/vignettes/pacehrh.rmd (hunk 1 above)
Comment (JSchripsema-IDM) [reply pacehrh/vignettes/pacehrh.rmd:42]: Oops! Good catch--I just added it.

**Review by MeWu-IDM — APPROVED** [review]

**Review by celiot-IDM — APPROVED** [review]
Sorry for the delay in approving!

**Pull request merged**